### PR TITLE
Feature/zip package python

### PR DESCRIPTION
**Pull request by poornachandra**

### CODE REVIEW

**Comment by poornachandra** [comment]
@sreevatsanraman I'm not sure about the deploy plugin though.

File: cdap-stream-clients/pom.xml [hunk 1]
```diff
@@ -34,5 +34,89 @@
...
     <module>java</module>
   </modules>
-
+  <profiles>
+    <profile>
+      <id>dist</id>
+      <properties>
+        <python.dir>${project.build.directory}/cdap-stream-clients</python.dir>
+        <python.stage.dir>${python.dir}</python.stage.dir>
+        <deploy.url>http://wwww.com</deploy.url>
```
Comment (sreevatsanraman) [inline cdap-stream-clients/pom.xml]: https://repository.continuuity.com/service/l
ocal/staging/deploy/maven2

**Comment by sreevatsanraman** [comment]
one comment. fix and merge

File: cdap-stream-clients/pom.xml [hunk 2]
```diff
@@ -34,5 +34,90 @@
...
+                <goals>
+                  <goal>deploy-file</goal>
+                </goals>
+                <configuration>
+                  <version>${project.version}</version>
+                  <groupId>${project.groupId}</groupId>
+                  <artifactId>python-stream-client</artifactId>
+                  <packaging>zip</packaging>
+                  <generatePom>false</generatePom>
+                  <file>${project.build.directory}/python-stream-client-${project.version}.zip</file>
```
Comment (sreevatsanraman) [inline cdap-stream-clients/pom.xml]: cdap-python-stream-client? Every other package has it. PLease change.

File: cdap-stream-clients/python/streamclient/v3/test/cdap_ssl_confi.json [hunk 3]
```diff
@@ -1,8 +0,0 @@
-{
```
Comment (anwar6953) [inline cdap-stream-clients/python/streamclient/v3/test/cdap_ssl_confi.json:1]: rename

Comment (anwar6953) [inline cdap-stream-clients/python/streamclient/v3/test/cdap_ssl_confi.json:1]: nvm, file was deleted.

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**